3.Promises-draw-circle/draw-circle-task-3/script.js

```diff
@@ -1,16 +1,24 @@
+function setWidthAndHeight(element, sizePx) {
+  element.style.height = `${sizePx}px`;
+  element.style.width = `${sizePx}px`;
+}
+
+function makeSquare(element, sizePx, radius) {
+  element.style.width = radius * sizePx + "px";
+  element.style.height = radius * sizePx + "px";
+}
+
 function showCircle(cx, cy, radius) {
   let div = document.createElement("div");
-  div.style.width = 0;
-  div.style.height = 0;
+  setWidthAndHeight(div, 0);
   div.style.left = cx + "px";
   div.style.top = cy + "px";
   div.className = "circle";
   document.body.append(div);
 
   return new Promise((resolve) => {
     setTimeout(() => {
-      div.style.width = radius * 2 + "px";
-      div.style.height = radius * 2 + "px";
+      makeSquare(div, 2, radius);
```

Comment:
```js
const size = radius * 2;
setWidthAndHeight(div, size);
```

by the way, compare with 
```js
setWidthAndHeight(div, radius * 2);
```
here you have to think what `radius * 2` means. The first option explicitly says that it is a size.

Reply:
now I see) Thank you!